src/javascript/binary/websocket_pages/trade/price.js

```diff
@@ -27,8 +28,99 @@ const formatMoney          = require('../../common_functions/currency').formatMo
  * `Price.display()` to display the price details returned from server
  */
 const Price = (() => {
+    
+
     let type_display_id_mapping = {};
     let form_id                 = 0;
+    let Highcharts;
+    const sparkline_options = {
```

Comment:
is it worth putting this config in another file? it doesn't really have anything to do with price.js